### DISCUSSION TITLE
Automatically add " when breaking long strings

### DISCUSSION THREAD

**Discussion by xxc-zsz** [microsoft/pylance-release#4155] [Enhancement]
Is there an extension that allows you to automatically break long strings correctly after pressing enter in the middle of it?
Eg:

`"ahahahahahahahaahahahahahahahahahah"`

Desired output: 
```
"ahahahahahahah"
"ahahahahahahahah"
```
Current output: 
```
"ahahahahahahah 
ahahahahahahahah"
```
Pycharm has been doing this for ages but NO WAY I can get VSCode to do the same. It's very frustrating at the moment

**Pycharm**
![pycharm](https://user-images.githubusercontent.com/108051416/227821872-55e7805c-6d77-4e1e-82be-0cd8a7bc98be.gif)

**vscode**
![vscode](https://user-images.githubusercontent.com/108051416/227821875-4d606b6b-9b63-4df4-a807-45b07894233c.gif)

**Comment by nikky-telus** [comment]
Yes please!

Absolute filepaths are horrible to work with without this.

**Comment by starball5** [comment]
Related on Stack Overflow: [How can I auto-insert a linebreak ("\") in vscode?](https://stackoverflow.com/q/77753534/11107541)

**Comment by luabud** [comment]
I changed the title of this discussion to make it clearer for the team but let me know if you don't think it represents well the ask. Thanks :)

**Comment by hww-nikhilshankarnoble** [comment]
Yes please! One of the things I've surprisingly missed the most transitioning from PyCharm -> VSCode

**Comment by jorimp** [comment]
I have been looking for a solution to this for ages. Many of my co-workers will not leave Pycharm for vscode simply because of this issue.

**Comment by Andrej730** [comment]
I believe it was implemented after #6961 and the feature is currently enabled by default (61e8983).

**Reply by aliksimon** [reply]
That sounds exactly what is needed. This settings also turned on in my VS Code but the quotes are not added. So the issue is still there.
